backend/exchanges/exchanges.go

```diff
@@ -236,7 +239,7 @@ func GetExchangeDeals(account accounts.Interface, regionCode string, action Exch
 		bestDealIndex := 0
 		for i, deal := range deals {
 			oldBestDeal := deals[bestDealIndex]
-			if deal.Fee < oldBestDeal.Fee {
+			if !deal.IsHidden && deal.Fee < oldBestDeal.Fee {
```

Comment:
[out of scope for this PR] I still think that if we keep the "best" badge we need to decide how to deal with multiple deals offering the same fee